apisix/plugins/limit-count/limit-count-redis-cluster.lua

```diff
@@ -0,0 +1,141 @@
+--
+-- Licensed to the Apache Software Foundation (ASF) under one or more
+-- contributor license agreements.  See the NOTICE file distributed with
+-- this work for additional information regarding copyright ownership.
+-- The ASF licenses this file to You under the Apache License, Version 2.0
+-- (the "License"); you may not use this file except in compliance with
+-- the License.  You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+
+local rediscluster = require("resty.rediscluster")
+local core = require("apisix.core")
+local resty_lock = require("resty.lock")
+local assert = assert
+local error = error
+local setmetatable = setmetatable
+local tostring = tostring
+local ipairs = ipairs
+
+local _M = {}
+
+local mt = {
+    __index = _M
+}
+
+
+local function split(inputstr, sep)
+    if sep == nil then
+       sep = "%s"
+    end
+
+    local t={}
+    for str in string.gmatch(inputstr, "([^"..sep.."]+)") do
+       table.insert(t, str)
+    end
+
+    return t
+ end
+
+
+-- https://github.com/steve0511/resty-redis-cluster
+local function new_redis_cluster(conf)
+    local config = {
+        name = "apisix-rediscluster",
```

Comment:
![image](https://user-images.githubusercontent.com/2106987/95658362-67185500-0b4c-11eb-8ac9-60aec4ee1c10.png)

Should we use a uniform name?